common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -927,6 +927,15 @@ private AuthenticationConstants() {
          */
         public static final String CALLER_INFO_PACKAGE = "caller.info.package";
 
+        /**
+         * String to send Msal V2 Request params.
+         */
+        public static final String BROKER_REQUEST_V2 = "broker.request.v2";
```

Comment:
Add a todo to say that we need to find a common place for this that does not say ADAL in the path